site/en/docs/extensions/reference/commands/index.md

```diff
@@ -8,89 +8,159 @@ You must have a `"manifest_version"` of at least `2` to use this API.
-not permitted in order to avoid conflicts with the AltGr key. Shift can be used in addition to Alt
-or Ctrl, but is not required. Modifiers (such as Ctrl) can not be used in combination with the Media
-Keys. Tab key was removed from list of supported keys in Chrome version 33 and above for
-accessibility reasons.
+`suggested_key`
 
-\* Please note that on Mac 'Ctrl' is automatically converted to 'Command'. If you want 'Ctrl'
-instead, please specify 'MacCtrl' under `"mac"`. Specifying 'MacCtrl' under `"default"` will cause
-the extension to be uninstallable.
+: An optional property that declares default keyboard shortcuts for the command. If omitted, the
+command will be unbound. This property can either take a string or an object value.
 
-\* Additionally, on Chrome OS, you can specify 'Search' as a modifier.
+: A string values specifies the default keyboard shortcut that should be used across all platforms.
 
-Certain Chrome shortcuts (e.g. window management) always take priority over Extension Command
-shortcuts and can not be overwritten.
+: An object value allows the extension developer to customize the keyboard shortcut for each
+platform. When providing platform-specific shortcuts, valid object properties are `default`,
+`chromeos`, `linux`, `mac`, and `windows`.
+
+: See [Key combination requirements][header-key-combos] for
+additional details.
+
+`description`
+
+: A string used to provide the user with a short description of the command's purpose. This string
+appears in extension keyboard shortcut management UI. Descriptions are required for standard
+commands, but are ignored for [Action commands][header-action].
+
+An extension can have many commands, but may specify at most 4 suggested keyboard shortcuts. The
+user can manually add more shortcuts from the `chrome://extensions/shortcuts` dialog.
+
+### Supported Keys
+
+The following keys are usable command shortcuts. Key definitions are case sensitive. Attempting to
+load an extension with an incorrectly cased key will result in a manifest parse error at
+installation time.
+
+Alpha keys
+
+: `A` … `Z`
+
+Numeric keys
+
+: `0` … `9`
+
+Standard key strings
+
+: General–`Comma`, `Period`, `Home`, `End`, `PageUp`, `PageDown`, `Space`, `Insert`, `Delete`
+
+: Arrow keys–`Up`, `Down`, `Left`, `Right`
+
+: Media Keys–`MediaNextTrack`, `MediaPlayPause`, `MediaPrevTrack`, `MediaStop`
+
+Modifier key strings
+
+: `Ctrl`, `Alt`, `Shift`, `MacCtrl` (macOS only), `Command` (macOS only), `Search` (Chrome OS only)
+
+{% Aside 'note' %}
+
+`Tab` was removed from list of supported keys in Chrome 33 for accessibility reasons.
+
+{% endAside %}
+
+### Key combination requirements {: #key-combinations }
+
+- Extension Command shortcuts must include either `Ctrl`\* or `Alt`.
+
+    - Modifiers **cannot** be used in combination with Media Keys.
+
+- \* On macOS `Ctrl` is automatically converted into `Command`.
```

Comment:
The `\*` seems to be cruft

Reply:
It is a carry-over from the original copy. It seems that the intent was to indicate that `Ctrl` behavior is modified on macOS, which is described in the next sibling bullet. I'd like to do some additional testing to verify modifier key requirements before deciding whether or not to remove this.

Reply:
After reading this a few more times I think it's probably safe to remove the asterisk. Doing so now.